Library/Homebrew/requirements/language_module_requirement.rb

```diff
@@ -12,11 +12,23 @@ def initialize(language, module_name, import_name = nil)
 
   satisfy(:build_env => false) { quiet_system(*the_test) }
 
-  def message; <<-EOS.undent
-    Unsatisfied dependency: #{@module_name}
-    Homebrew does not provide #{@language.to_s.capitalize} dependencies; install with:
-      #{command_line} #{@module_name}
+  def message
+    s = ""
+
+    s << <<-EOS.undent
```

Comment:
Thanks Xu. Should have occurred to me really.

Just trying to work out why this error only gets thrown sometimes and then I'll push again:

```
  1) Error:
LanguageModuleRequirementTests#test_differing_module_and_import_name:
NoMethodError: undefined method `exitstatus' for nil:NilClass
    /usr/local/Library/Homebrew/requirements/language_module_requirement.rb:24:in `message'
    /usr/local/Library/Homebrew/test/test_language_module_requirement.rb:26:in `test_differing_module_and_import_name'
```